src/index.ts

```diff
@@ -209,6 +209,7 @@ async function runUpgrades(options: Options, timeout?: NodeJS.Timeout): Promise<
         ...rcConfig,
         packageFile: packageInfo.filepath,
         workspacePackages,
+        errorLevel: 1,
```

Comment:
Why set `errorLevel` here?

Reply:
if we do not, it will throw if there are upgrades, which is the current behaviour we are trying to fix

Reply:
Okay, thanks. It seems wrong that we have to hardcode this here when the default value of `errorLevel` is already 1. Maybe we can consolidate the workspace and non-workspace logic to avoid this? 

I will need to take a closer look at how it's handled to know. I'm not available as much right since I'm traveling, but I can check it out soon.

Reply:
the default is indeed 1, but in my use case, we pass 2, thus causing it to exist on the first workspace that needs updating. This will ensure that it will not error on first workspace.
If you want, we can instead pass a flag to runLocal that instructs to not exit on upgrades needed. What do you think?

Reply:
If we need to override the  errorLevel passed to `runLocal`, then the exit logic is probably in the wrong place. Without having looked at the code yet, I'm thinking we need to return early from `runLocal` but only exit the process at the end.

Reply:
I think I see your point - I will make the change

Reply:
Done - cleaner code now